src/client/feed/feedReducer.js

```diff
@@ -0,0 +1,12 @@
+import { UPDATE_FEED_RESULTS } from '../search/SearchActions';
+
+const feed = (state = { 'posts': [] }, action) => {
+  switch (action.type) {
+    case UPDATE_FEED_RESULTS:

```

Comment:
When you say defined are you talking about the constant UPDATE_FEED_RESULTS ? I agree that should be in feedActions as is right now, or rather searchActions. 

I'm talking about the case itself, I don't think there's any reason this should be in a new file and feel this should be part of the searchReducer since it's performing a search, the feed stuff would be interactions and actions on the actual feed in my mind. Where as these are actions to do a search.